LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 DevOpsDays
+Copyright (c) 2017 devopsdays
```

Comment:
I think that it's actually supposed to be `Copyright (c) 2016-2017` since we started this code in 2016; by putting only 2017 we have a prior art problem. I don't think it's a huge deal, but I'd rather cover the years of development.